contributing_to_docs/doc_guidelines.adoc

```diff
@@ -484,6 +484,39 @@ $ lorem.sh
 ----
 ....
 
+* Do not use more than one command per code block. For example, the following should be split up into three separate code blocks.
```

Comment:
But is it _should_ or _must_?

Reply:
Good point, @jboxman. I can change it to "must" since this is the guideline moving forward. @bobfuru, thoughts?

Reply:
Good catch; my vote is _must_.

Reply:
Seconding "must" as an RFC 2119 booster. ISG agrees, too.